protocol/v2/ssv/runner/attester.go

```diff
@@ -181,7 +183,8 @@ func (r *AttesterRunner) ProcessPostConsensus(logger *zap.Logger, signedMsg *spe
 
 		logger.Info("✅ successfully submitted attestation",
 			zap.String("block_root", hex.EncodeToString(signedAtt.Data.BeaconBlockRoot[:])),
-			fields.ConsensusTime(time.Since(r.started)),
+			fields.ConsensusTime(consensusDuration),
+			zap.Float64("submission_time", time.Since(start).Seconds()),
```

Comment:
This is likely not the only place where we want to log submission time, so shall we create a function for this in the `fields` package?

Reply:
@nkryuchkov done